hail_search/constants.py

```diff
@@ -10,6 +10,16 @@
 
 VARIANT_KEY_FIELD = 'variantId'
 GNOMAD_GENOMES_FIELD = 'gnomad_genomes'
+SPLICE_AI_FIELD = 'splice_ai'
+NEW_SV_FIELD = 'new_structural_variants'
+SCREEN_KEY = 'SCREEN'
```

Comment:
Does `SCREEN` need to be all caps?

Reply:
yeah it was a weird choice we made in the UI. If it really bothers you we could transform it in the server side proxy